css/estilos-index.css

```diff
@@ -0,0 +1,196 @@
+@import url(flexboxgrid.min.css);
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;600;700&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+}
+#enlace-logo{
+    width: 0;
+    float: left;
+    padding: 0;
+    margin: 0 0 0 0;

```

Comment:
Reduce the arguments to a single unit as the margins on every sides is the same amount.